docs/get_started/tour.mdx

```diff
@@ -1540,15 +1569,16 @@ Let's recap what you did! You have built a ticket reservation system that is res
-| What you implemented                                        | What you didn't implement, as Restate handles it for you                                       |
-|--------------------------------------------------------------|------------------------------------------------------------------------------------------------|
-| ✅ Request-response invocations                               | ❌ Handling retries, timeouts, etc.                                                            |
-| ✅ Sending messages                                           | ❌ Deploy and operate message queues for async requests                                        |
+| What you implemented                                                     | What you didn't implement, as Restate handles it for you                                       |
+|--------------------------------------------------------------------------|------------------------------------------------------------------------------------------------|
+| ✅ Request-response invocations                                           | ❌ Handling retries, timeouts, etc.                                                            |
+| ✅ Sending messages                                                       | ❌ Deploy and operate message queues for async requests                                        |
+| ✅ Idempotent HTTP calls                                        | ❌ Write deduplication logic                                       |
 | ✅ Durable Execution: retries, partial progress recovery, and suspensions | ❌ Manual retry logic and partial progress recovery                                 |
-| ✅ Durable timers: sleeping and scheduling async tasks         | ❌ Workflow orchestrators or cron jobs for scheduling tasks                                     |
-| ✅ Virtual Objects: concurrency guarantees and shared state    | ❌ Guards for keeping state consistent across retries, concurrent requests, and scaling out.    |
-| ✅ K/V state: storing and inspecting                          | ❌ Session databases for state. State consistency guards.                                       |
-| ✅ Storing computation results in the journal                 | ❌ Logic to make operations idempotent (e.g. generate idempotency keys)                         |
+| ✅ Durable timers: sleeping and scheduling async tasks                    | ❌ Workflow orchestrators or cron jobs for scheduling tasks                                     |
+| ✅ Virtual Objects: concurrency guarantees and shared state               | ❌ Guards for keeping state consistent across retries, concurrent requests, and scaling out.    |
+| ✅ K/V state: storing and inspecting                                      | ❌ Session databases for state. State consistency guards.                                       |
+| ✅ Storing computation results in the journal                             | ❌ Logic to make operations idempotent (e.g. generate idempotency keys)                         |
```

Comment:
I really like this summary Giselle. It could even work w/o having gone through the whole tour in the sense of telling people what Restate does for them on a finer-grained level.